plugins/tencent/README.md

```
@@ -0,0 +1,97 @@
+1. **Start the agent**:
+
+   ```bash
+   cd plugins/tencent
+   docker compose run --rm tencent-agent
+   ```
+
+   The `test-client` service starts automatically as a dependency and serves `example/` on `http://localhost:8000/`. It stays up between agent runs; stop it explicitly with `docker compose down` when done.
+
+   On join the agent logs a clickable URL like:
+
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)